account-kit/smart-contracts/src/6900-v08-RI/modules/single-signer-validation/module.ts

```diff
@@ -1,6 +1,7 @@
 import type { Address } from "viem";
 
 const addresses = {
+  1337: "0x9DA8c098A483E257dd96022831DF308cB24fCBE6",
   421614: "0x9DA8c098A483E257dd96022831DF308cB24fCBE6",
 } as Record<number, Address>;
```

Comment:
for LA, we have this as an entry of `Record<number | "default", address>` and use the number field as an override for the addresses that are the same. That way we always can get address as `addresses[chain.id] ?? address.default` so that we don't have to update the SDK when we deploy a new chain

Reply:
Makes sense, updated!